backend-server/src/main/java/org/dice/ida/util/RDFUtil.java

```diff
@@ -320,20 +329,39 @@ public String getVizIntent(String viz) {
 				"  ?param rdfs:label ?paramLabel ." +
 				"  ?param ivodp:isOptional ?isoptional  " +
 				"}";
-		ResultSet attributeResultSet = getResultFromQuery(queryString);
+		ResultSet attributeResultSet = getResultFromQuery(queryString, "ida_viz");
 		if (attributeResultSet == null) {
 			return null;
 		}
 		while (attributeResultSet.hasNext()) {
 			QuerySolution querySolution = attributeResultSet.next();
 			String param = querySolution.get("paramLabel").asLiteral().getString();
 			boolean optional = (boolean) querySolution.get("isoptional").asNode().getLiteralValue();
-			attributeOptionalMap.put(param,optional);
+			attributeOptionalMap.put(param, optional);
 		}
 		if (conn != null) {
 			conn.close();
 		}
 		model = null;
 		return attributeOptionalMap;
 	}
+
+	public String addDatasetName(String dsName) {
+		String queryString = "PREFIX ab: <https://www.upb.de/ida/datasets/>" +
+				" INSERT DATA {" +
```

Comment:
Prone to injections, please look for a better way to do this.

Reply:
taken care of